source/extensions/filters/network/http_connection_manager/config.h

```diff
@@ -264,6 +265,8 @@ class HttpConnectionManagerConfig : Logger::Loggable<Logger::Id::config>,
   static const uint64_t RequestTimeoutMs = 0;
   // request header timeout is disabled by default
   static const uint64_t RequestHeaderTimeoutMs = 0;
+  Http::PathTransformer forwarding_path_transformer_;
+  Http::PathTransformer filter_path_transformer_;
```

Comment:
Are these being used somewhere?

Reply:
No, I am wondering how they can be compatible with the current interface or how to deprecate the current interface.